packages/@sanity/portable-text-editor/src/types/editor.ts

```diff
@@ -282,65 +288,122 @@ export type OnPasteResultOrPromise = OnPasteResult | Promise<OnPasteResult>
 export type OnPasteFn = (arg0: {
   event: React.ClipboardEvent
   path: Path
-  portableTextFeatures: PortableTextFeatures
-  type: ArraySchemaType<PortableTextBlock>
+  types: PortableTextMemberTypes
   value: PortableTextBlock[] | undefined
 }) => OnPasteResultOrPromise
 
 export type OnBeforeInputFn = (event: Event) => void
 
 export type OnCopyFn = (
   event: React.ClipboardEvent<HTMLDivElement | HTMLSpanElement>
-) => undefined | any
+) => undefined | unknown
 
 export type PatchObservable = Observable<{
   patches: Patch[]
   snapshot: PortableTextBlock[] | undefined
 }>
 
-export type RenderAttributes = {
-  annotations?: PortableTextBlock[]
+/** @public */
```

Comment:
`@beta`?

Reply:
might be good to start out with @beta yeah